### DISCUSSION TITLE
How to integrate SQLAlchemy in robyn

### DISCUSSION THREAD

**Discussion by iakirago** [sparckles/Robyn#471] [Q&A] [answered]
How to integrate SQL Alchemy in robyn?

**Comment by sansyrox** [comment]
@iakirago , I created this demo app to showcase the use of sqlite with robyn - https://github.com/sansyrox/robyn-demo/blob/main/api.py

Can do something similar for sqlalchemy. I am planning to write a tutorial soon for it 😄

**Reply by iakirago** [reply, marked as answer]
Thank you so much. If you have completed the tutorial, please leave me a message under this question.
